staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/namer.go

```diff
@@ -37,6 +37,11 @@ func (n localNamer) Name(t *types.Type) string {
 	if len(n.localPackage.Package) != 0 && n.localPackage.Package == t.Name.Package {
 		return t.Name.Name
 	}
+	// For non-local and non-fundamental types, use an absolute reference
+	// see https://protobuf.com/docs/language-spec#type-references
+	if strings.Contains(t.Name.Package, ".") {
+		return fmt.Sprintf(".%s", t.Name.String())
```

Comment:
Wouldn't `t.Name` be enough? 

```suggestion
		return fmt.Sprintf(".%s", t.Name)
```

Reply:
Brought this change in, thanks!